pkg/cloudserver/websockets.go

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) The One True Way 2022. Apache License 2.0. The authors accept no liability, 0 nada for the use of this software.  It is offered "As IS"  Have fun with it!!
+ */
+
+package cloudserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+	"github.com/nats-io/nats.go"
+	log "github.com/sirupsen/logrus"
+	"github.com/theotw/natssync/pkg"
+	"github.com/theotw/natssync/pkg/bridgemodel"
+	v1 "github.com/theotw/natssync/pkg/bridgemodel/generated/v1"
+	"github.com/theotw/natssync/pkg/msgs"
+)
+
+const (
+	hostAddress = "server"
+)
+
+func client() error {
+	hostUrl := url.URL{
+		Scheme: "https",
+		Host:   hostAddress,
+		Path:   "/",
+	}
+	conn, _, err := websocket.DefaultDialer.Dial(hostUrl.String(), nil)
+	if err != nil {
+		return err
+	}
+
+	msg := []byte("Hello, world")
+	err = conn.WriteMessage(websocket.TextMessage, msg)
+	if err != nil {
+		return err
+	}
+
+	_, msg, err = conn.ReadMessage()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(string(msg))
+	return nil
+}
```

Comment:
I think this code was a copy and paste from my POC work, which can be deleted. I think I was just using it for testing.